src/unwinder/arch/loongarch64.rs

```diff
@@ -0,0 +1,228 @@
+use core::{fmt, ops};
+use gimli::{LoongArch, Register};
+
+use super::maybe_cfi;
+
+pub const MAX_REG_RULES: usize = 74;
+
+#[repr(C)]
+#[derive(Clone, Default)]
+pub struct Context {
+    pub gp: [usize; 32],
+    pub fp: [usize; 32],
```

Comment:
This could be optional depending on target_feature? See riscv64.

Is the idea to avoid having different size of `Context` to simplify assembly?